zipkin/src/main/scala/org/http4s/zipkin/examples/Config.scala

```diff
@@ -0,0 +1,7 @@
+package org.http4s.zipkin.examples
```

Comment:
Our examples usually go in the examples directory up top, so they don't contribute to bloat of the main code.